components/tool/taskManager/src/domain/config.js

```diff
@@ -8,9 +10,9 @@ export default class Config {
     this._config = {
       AVAILABLE_STATUS: {
         CANCELLED: 'CANCELLED',
-        COMPLETED: 'COMPLETED 🥳 ',
+        COMPLETED: <ChevronCircleDownSolid size="medium" />,
```

Comment:
🍆 This is a bit weird, as domain config should be decoupled from react things. However, could be a tradeoff maybe...

Reply:
agree! in next commit i will refactor this